ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesImageSinglePage.tsx

```diff
@@ -1,9 +1,205 @@
+            )}
+            {scan && (
+                <Label isCompact>
+                    Scan time: {getDateTime(scan.scanTime)} by {scan.dataSource.name}
+                </Label>
+            )}
+            {sha && (
+                <Tooltip content="Copy image SHA to clipboard">
+                    <Label
+                        style={{ cursor: 'pointer' }}
+                        icon={<CopyIcon />}
+                        isCompact
+                        color={hasSuccessfulCopy ? 'green' : 'grey'}
+                        onClick={() => copyToClipboard(sha)}
+                    >
+                        {hasSuccessfulCopy ? 'Copied!' : 'SHA'}
+                    </Label>
+                </Tooltip>
+            )}
+        </LabelGroup>
+    );
+}
+
 function WorkloadCvesImageSinglePage() {
     const { imageId } = useParams();
-    return <>Workload CVE Image Single Page: {imageId}</>;
+    const { data, error } = useQuery<ImageDetailsResponse, ImageDetailsVariables>(
+        imageDetailsQuery,
+        {
+            variables: { id: imageId },
+        }
+    );
+
+    const [activeTabKey, setActiveTabKey] = useDetailsTabParameter();
+
+    const imageData = data && data.image;
+    const imageName = imageData?.name?.fullName ?? 'NAME UNKNOWN';
+
+    const handleTabClick: TabsProps['onSelect'] = (e, tabKey) => {
+        if (isDetailsTab(tabKey)) {
+            setActiveTabKey(tabKey);
+        }
+    };
+
+    let mainContent: ReactNode | null = null;
+
+    if (error) {
+        mainContent = (
+            <PageSection variant="light">
+                <Bullseye>
+                    <EmptyState variant={EmptyStateVariant.large}>
+                        <EmptyStateIcon
+                            className="pf-u-danger-color-100"
+                            icon={ExclamationCircleIcon}
+                        />
+                        <Title headingLevel="h2">{getAxiosErrorMessage(error)}</Title>
+                    </EmptyState>
+                </Bullseye>
+            </PageSection>
+        );
+    } else {
+        mainContent = (
+            <>
+                <PageSection variant="light">
+                    {imageData ? (
+                        <Flex direction={{ default: 'column' }}>
+                            <Title headingLevel="h1" className="pf-u-mb-sm">
+                                {imageName}
+                            </Title>
+                            <ImageDetailBadges imageData={imageData} />
+                        </Flex>
+                    ) : (
+                        <Flex
+                            direction={{ default: 'column' }}
+                            spaceItems={{ default: 'spaceItemsXs' }}
+                            className="pf-u-w-50"
+                        >
+                            <Skeleton fontSize="2xl" />
+                            <Skeleton fontSize="sm" />
```

Comment:
ooh fancy. is this the way we're planning on handling loading sections now instead of spinner/loaders?